src/lib/permissions.ts

```diff
@@ -26,3 +32,9 @@ export const ADMIN_PERMS: ApplicationCommandPermissionData = {
 	permission: true,
 	type: 'ROLE'
 };
+
+export const BOTMASTER_PERMS: ApplicationCommandPermissionData = {
+	id: ROLES.ADMIN,
+	permission: true,
+	type: 'ROLE'
+};
```

Comment:
What's the point of this? It's the same as ADMIN_PERMS

Reply:
I think this was meant to be perms for those only on the team, rather than those with the purple admin role. I suggested in prune.ts (before slashing commands) that its perms be changed to that so if anyone wreaks havoc it's pretty much protected

Reply:
temp solution